components/NavigationDrawer/src/private/MDCBottomDrawerContainerViewController.m

```diff
@@ -628,6 +660,11 @@ - (void)updateContentHeaderWithTransitionToTop:(CGFloat)headerTransitionToTop
                                : self.contentHeaderTopInset - headerTransitionToTop * headersDiff;
   contentHeaderView.frame =
       CGRectMake(0, contentHeaderViewTop, contentHeaderViewWidth, contentHeaderViewHeight);
+  self.shadowedView.frame = contentHeaderView.frame;
+  if (self.headerViewController.view.layer.mask) {
+    self.shadowedView.layer.shadowPath =
+        ((CAShapeLayer *)self.headerViewController.view.layer.mask).path;
```

Comment:
Same question/feedback about explicit casting. I believe it's either unnecessary or should be preceded with a class check.  I understand that clients control `headerViewController` and could potentially set any `CALayer` subclass.

Reply:
Done.